lib/cocoapods/installer/xcode/pods_project_generator.rb

```diff
@@ -151,7 +151,6 @@ def prepare
               build_configuration.build_settings['STRIP_INSTALLED_PRODUCT'] = 'NO'
               build_configuration.build_settings['CLANG_ENABLE_OBJC_ARC'] = 'YES'
               build_configuration.build_settings['CODE_SIGNING_REQUIRED'] = 'NO'
-              build_configuration.build_settings['PROVISIONING_PROFILE_SPECIFIER'] = 'NO_SIGNING/' # a bogus provisioning profile ID assumed to be invalid
```

Comment:
i think we might also want to set `CODE_SIGNING_ALLOWED = NO` ?

Reply:
That's a reasonable proposal IMO, but please consider doing that outside of this PR.  A lot of folks are waiting for this and I'd rather not drag it on with additional discussions about other fields.

Reply:
After talking with some folks, I'm almost certain that setting _needs_ to be set in order to have Xcode not try to code sign at all, which is why we added the bogus provisioning profile in the first place.

Reply:
If that were the case then people's workarounds wouldn't be working (clearing just the provisioning profile specifier).